test-data/mini_ert/jobs/perlin.py

```diff
@@ -534,4 +534,4 @@ def readParameters(filename):
                 x = i / 8.0
                 obs = observation.perlinNoise1D(x)
                 noise = report_step_noise.perlinNoise1D(x)
-                f.write("%f\n" % (obs + offset + noise * scale))
+                f.write(f"{obs + offset + noise * scale}\n")
```

Comment:
Here, as this is the result of equation, I would use some casting to float, hm?

Reply:
I don't think it is necessary, (at least some of) these variables are already float, to the expression is evaluated to a float, before the `f`-string kicks in and converts the float to a string.